mlflow/store/sqlalchemy_store.py

```diff
@@ -202,6 +204,7 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
                     self._list_experiments(session=session, view_type=view_type)]
 
     def _get_experiment(self, session, experiment_id, view_type):
+        experiment_id = experiment_id or SqlAlchemyStore.DEFAULT_EXPERIMENT_ID
         experiments = self._list_experiments(
             session=session, ids=[experiment_id], view_type=view_type).all()
```

Comment:
Does this work? Isn't experiment_id a string, but expects an int with respect to the sqlalchemy store? Is there a test that proves this behavior?

Reply:
It does. The only call that needs experiment_id to be int is _list_experiments. It works because it is cast to in on L193 in _list_experiments